### PR TITLE
10 delete business

### PR DESCRIPTION
**Pull request by StephenZeiler**
This PR focuses on allowing the user to delete a business. 

Changes made: 
-Logic to allow the user to delete a business.
-Success/Failure message corresponding to the http response code.

### CODE REVIEW

File: react-locate-contractor/src/config.tsx [hunk 1]
```diff
@@ -3,14 +3,24 @@ export default {
         issuer: 'https://dev-81535986.okta.com/oauth2/default',
         clientId: '0oa5se2c6gLXwYT2S5d7',
         scopes: ['openid', 'profile', 'email'],
+<<<<<<< HEAD
```
Comment (Krescenskok) [inline react-locate-contractor/src/config.tsx]: Don't forget to take out conflict comments

File: react-locate-contractor/src/Services/GetBusiness.tsx [hunk 2]
```diff
@@ -23,6 +22,20 @@ function GetBusinessData(searchString: any): JSX.Element {
             })
     }, [])
     const handleEditBusiness = () => setEditMode(true);
+    const handleDeleteBusiness = () => {
+        setDeleteMode(false)
+        deleteBusiness(searchString.searchString)
+            .then((res) => {
+                console.log(res.status)
```
Comment (Krescenskok) [inline react-locate-contractor/src/Services/GetBusiness.tsx]: Remove console log

Comment (StephenZeiler) [reply react-locate-contractor/src/Services/GetBusiness.tsx]: removed

File: react-locate-contractor/src/config.tsx [hunk 3]
```diff
@@ -3,14 +3,24 @@ export default {
...
+        redirectUri: `http://localhost:3000/login/callback`
+        // `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`
+=======
         redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`
         // redirectUri: `http://localhost:3000/login/callback`
+>>>>>>> main
```
Comment (Krescenskok) [inline react-locate-contractor/src/config.tsx]: You still have multiple merge comments

Comment (StephenZeiler) [reply react-locate-contractor/src/config.tsx]: I thought I took care of them, they should be gone now.

File: API-LocateContractor/Controllers/BusinessController.cs [hunk 4]
```diff
@@ -92,7 +92,10 @@ public ObjectResult Delete(string userEmail)
                     _businessContext.SaveChanges();
                     return StatusCode(StatusCodes.Status200OK, business);
                 }
-                return StatusCode(StatusCodes.Status304NotModified, business);
+                else
+                {
+                    return StatusCode(StatusCodes.Status304NotModified, business);
```
Comment (Dwernery) [inline API-LocateContractor/Controllers/BusinessController.cs]: Don't want to use a 304 here. that's for cached files that can be used when the underlying resource has not been modified

**Review by Dwernery — CHANGES_REQUESTED** [review]

**Review by Dwernery — APPROVED** [review]

**Pull request merged**